crates/bitwarden-vault/src/collection.rs

```diff
@@ -40,6 +42,7 @@ pub struct CollectionView {
 }
 
 impl LocateKey for Collection {
+    /// Returns a [SymmetricCryptoKey] on success, [CryptoError] on failure
```

Comment:
This comment conveys the exact same information as the type interface. We generally want to avoid these type of comments since they can easily get out of sync with the actual code. We're also actively refactoring these logic.